x-pack/plugins/apm/server/lib/transactions/charts/get_avg_response_time_anomalies/get_anomaly_aggs.js

```diff
@@ -63,9 +63,8 @@ export async function getAnomalyAggs({
     const resp = await client('search', params);
     return resp.aggregations;
   } catch (e) {
-    if (e.statusCode === 404) {
-      return null;
-    }
-    throw e;
+    // swallow error because there are lots of reasons
+    // the ml index lookup may fail
```

Comment:
I'm split on leaving it like this, or changing it to:
```js
if (e.statusCode >= 400 && e.statusCode < 500) {
  return null;
}
throw e;
```

Reply:
Yeah, blindly swallowing all errors might be a bit much. I think I like your suggestion with only swallowing 4xx errors.

A middle ground could be swallowing all http errors:
```
if (e.statusCode > 0) {
  return null;
}
throw e;
```